### PR TITLE
mlx90393 support added

### LINKED ISSUE #6000: Support Magnetometer MLX90393
## Current Behavior
Unsupported for now.

## Desired Behavior
Supported.

## Suggested Solution
Write support

## Who does this impact? Who is this for?
All who use Ublox (M9N). And wants to use Sparkfun compass.

## Additional context
Very good magnetometer. (M9N GPS also good to use with)
https://www.sparkfun.com/products/14571

### PR DESCRIPTION
**Pull request by Elkin-Vasily**
closes #6000

### CODE REVIEW

File: src/main/drivers/compass/compass_mlx90393.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * This file is part of Cleanflight.
```
Comment (JulioCesarMatias) [inline src/main/drivers/compass/compass_mlx90393.h]: change "Cleanflight" to "iNav"

File: src/main/drivers/compass/compass_mlx90393.c [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/*
+ * This file is part of Cleanflight.
```
Comment (JulioCesarMatias) [inline src/main/drivers/compass/compass_mlx90393.c]: change "Cleanflight" to "iNav"

File: src/main/drivers/compass/compass_mlx90393.c [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+// Burst Data Rate 0b000100
+#define TCMP_EN_REG_VALUE           0x62C4
+// Register 3
+#define RES_XYZ_OSR_FLT_REG         0x8 //from datasheet 0x2 << 2 = 0x8 
+// Oversampling 0b01
+// Filtering 0b111
+#define RES_XYZ_OSR_FLT_REG_VALUE   0x001D
+
+
+static void mlx_write_register(magDev_t * mag, uint8_t reg_val, uint16_t value) {
```
Comment (DzikuVx) [inline src/main/drivers/compass/compass_mlx90393.c]: Can we please use camelCase over here as well?

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/drivers/compass/compass_mlx90393.c (hunk 3 above)
Comment (Elkin-Vasily) [reply src/main/drivers/compass/compass_mlx90393.c]: Done

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**